internal/bminventory/inventory.go

```diff
@@ -962,14 +962,15 @@ func (b *bareMetalInventory) updateImageInfoPostUpload(ctx context.Context, clus
 	cluster.ImageInfo.SizeBytes = &imgSize
 
 	// Presigned URL only works with AWS S3 because Scality is not exposed
+	signedURL := ""
 	if b.objectHandler.IsAwsS3() {
-		signedURL, err := b.objectHandler.GeneratePresignedDownloadURL(ctx, imgName, imgName, b.Config.ImageExpirationTime)
+		signedURL, err = b.objectHandler.GeneratePresignedDownloadURL(ctx, imgName, imgName, b.Config.ImageExpirationTime)
```

Comment:
note to self: undo this. included in https://github.com/openshift/assisted-service/pull/1111